Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -154,6 +154,7 @@ Enter the choice (1|2)";
                 ($control.ControlId -eq "Azure_AppService_DP_Use_Secure_TLS_Version") -or
                 ($control.ControlId -eq "Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN") -or
                 ($control.ControlId -eq "Azure_APIManagement_DP_Use_HTTPS_URL_Scheme") -or
+                ($control.ControlId -eq "Azure_SQLDatabase_DP_Use_Secure_TLS_Version_Trial") -or
```

Comment:
We need to add BRS details in Scripts/RemediationScripts/Readme.md file as well.

Reply:
Added